packages/design-system-react-native/figma.config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "codeConnect": {
+    "include": ["packages/design-system-react-native/src/components/**/*.tsx"],
+    "exclude": ["**/temp-components/**/*"],
```

Comment:
Ignoring `temp-components` as we intentionally use it for non-Figma aligned components